Neos.RedirectHandler/Classes/Command/RedirectionCommandController.php

```diff
@@ -0,0 +1,165 @@
+<?php
+namespace Neos\RedirectHandler\Command;
+
+/*
+ * This file is part of the TYPO3.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use League\Csv\Reader;
+use League\Csv\Writer;
```

Comment:
Keep in mind that this package is in the dev collection, but optional in the Flow distribution. So dependency is not really important here. 

I will try to do my best to have this package with no Neos dependencies, so a generic redirection handling for Flow. But people a free to use it. The Gedmo and League package are quiet common in the PHP community (even if we can "not like" some parts).